src/Plugin.php

```diff
@@ -106,38 +108,6 @@ private function ingestFacadeStubs(
         $registration->addStubFile($cache_dir . 'stubs.stubphp');
     }
 
-    /**
-     * @param \Illuminate\Foundation\Application|\Laravel\Lumen\Application $app
-     * @param \Illuminate\View\Factory $view_factory
-     */
-    private function ingestMetaStubs(
```

Comment:
The meta files generated by ide-helper create a stub for `optional`, which prevents us from properly making our own stub. We can just remove the usage of this for now